era5_cloud/README.md

```diff
@@ -0,0 +1,11 @@
+# Cloud data, monthly means, 2010
+
+This artifact repackages data coming from [ERA5 reanalysis](https://cds.climate.copernicus.eu/datasets/reanalysis-era5-single-levels-monthly-means?tab=overview) 
+and contains monthly averaged cloud properties in 2010.
+
+The input file is defined in pressure coordinates. We convert this into altitude over mean-sea level
+using $P = P^*exp(-z / H)$ with scale height $H$. We assume $P^* = 1e5$ (Pa) and $H = 7000$ (m). The output 
+is a NetCDF file that contains cloud fraction, cloud liquid water content (kg/kg), and cloud ice water content (kg/kg) 
+defined on a lon-lat-z-time grid.
```

Comment:
I think we should add support for reading directly pressure levels at some point.